### DISCUSSION TITLE
Mobile browser  connect metaMask app， how to connect

### DISCUSSION THREAD

**Discussion by Vivienne-CPZ** [WalletConnect/walletconnect-monorepo#1759] [Developer Support]
Mobile browser connected to mateMask wallet

Always open the mateMask app without connect

How to call signature?

I use the same as the example
 https://github.com/WalletConnect/web3modal/tree/V2/examples/react
 
 desktop is ok

https://user-images.githubusercontent.com/104967115/207565550-d3b8880a-274f-439e-90bf-2b64c3662684.mov

mobile browser without connect

https://user-images.githubusercontent.com/104967115/207565047-97ad744c-2bdd-407a-8bf7-0241e6334dae.mp4

my code
https://github.com/Vivienne-CPZ/web3modal-test

**Comment by yanchain** [comment]
Have you solved it?
